src/component/styles/book.css

```diff
@@ -1,9 +1,216 @@
+@font-face {
+  font-family: 'robotoslab-bold';
+  src: url(fontfamily/RobotoSlab-Bold.ttf);
+}
+
+@font-face {
+  font-family: 'roboto-thin';
+  src: url(fontfamily/RobotoSlab-Thin.ttf);
+}
+
+@font-face {
+  font-family: 'roboto-light';
+  src: url(fontfamily/RobotoSlab-Light.ttf);
+}
+
+@font-face {
+  font-family: 'roboto-extralight';
+  src: url(fontfamily/RobotoSlab-ExtraLight.ttf);
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url(fontfamily/Montserrat-ExtraBold.ttf);
+}
+
+@font-face {
+  font-family: 'Montserrat-regular';
+  src: url(fontfamily/Montserrat-Regular.ttf);
+}
+
 .list-item {
   display: flex;
-  flex-direction: column;
   row-gap: 20px;
+  background-color: hsl(0, 0%, 100%);
+  border: solid 1px #e8e8e8;
+  border-radius: 4px;
+  padding: 2rem;
+  margin: 0.398rem 0;
+}
+
+.first {
+  width: 45%;
+}
+
+.booklist {
+  padding: 0 6%;
+}
+
+.booklist .borders {
+  padding-bottom: 2.5rem;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+.b-title {
+  font-size: 1.375rem;
+  font-weight: bold;
+  color: #121212;
+  font-family: 'robotoslab-bold', sans-serif;
+  margin-left: 0;
+}
+
+.category {
+  color: #6e7173;
+  font-family: 'Montserrat', serif;
+  font-weight: bold;
+  font-size: 0.875rem;
+}
```

Comment:
- [ ] As mentioned, please make sure the color of the category text matches the one specified in the design as it should be `#8a898a` and not `#6e7173`. It's a small difference but it will make your UI look more polished.